README.md

```diff
@@ -14,6 +14,7 @@ To set up the branch:
 1. Create a new folder and RStudio project just for this storm. Clone the GitHub repository into this project. Because it's a separate folder, you'll be able to add storm-specific data and image files to this folder/project without overwriting files from other storms. We recommend the folder and project name `vizstorm-GIF-[stormname-year]`.
 1. Create and switch to a new storm-specific branch. Use the branch naming convention `Stormname-year`, e.g., `Matthew-2016`.
 1. Edit .gitignore so that the following files are *not* ignored: `*.ind`, `build/status/*.yml`, `viz_config.yml`, `lib/cfg/gd_config.yml`, `2_process/src/filter_sites_custom.R`.
+1. Add a lib/cfg/gd_config.yml. See other storm branches and lib/cfg/gd_config-example.yml for examples.
```

Comment:
i think this is in line 21 (a few steps below) and belongs after the creation of the Drive folder. But I like your language here - feel free to replace/edit my language below to make it more concise.

Reply:
oh I guess I thought those were different things. It was just never said that `gd_config-example.yml` needed to be changed or copied to be `gd_config.yml` before it started talking about using `gd_config.yml`. I'll just clarify that in the existing step

Reply:
Oh nope, I get it now. Running `scipiper::gd_config(config_file="lib/cfg/gd_config.yml", folder="drivefolderid")` does actually create that file. 👍